protocol/src/index.ts

```diff
@@ -651,6 +651,21 @@ export namespace Protocol {
     '|controlshtml|': readonly ['controlshtml', HTML];
     '|fieldhtml|': readonly ['fieldhtml', HTML];
     '|debug|': readonly ['debug', Message];
+    '|deinit|': readonly ['deinit'];
+    '|selectorhtml|': readonly ['selectorhtml', string, HTML];
+    '|refresh|': readonly ['refresh'];
+    '|tempnotify|': readonly ['tempnotify', string, Message, ...[] | [Message] | [Message, string]];
+    '|tempnotifyoff|': readonly ['tempnotifyoff', string];
+    '|noinit|namerequired|': readonly ['noinit', 'namerequired', Message];
+    '|noinit|nonexistent|': readonly ['noinit', 'nonexistent', Message];
+    '|noinit|joinfailed|': readonly ['noinit', 'joinfailed', Message];
+    '|noinit|rename|': readonly ['noinit', 'rename', string, string];
```

Comment:
I think the first string arg here is a `RoomID` and the second one is a `RoomTitle`?